nixos/modules/services/backup/borgbackup.nix

```diff
@@ -41,14 +41,26 @@ let
         $extraInitArgs
       ${cfg.postInit}
     fi
-  '' + ''
-    borg create $extraArgs \
-      --compression ${cfg.compression} \
-      --exclude-from ${mkExcludeFile cfg} \
-      $extraCreateArgs \
-      "::$archiveName$archiveSuffix" \
-      ${escapeShellArgs cfg.paths}
-  '' + optionalString cfg.appendFailedSuffix ''
+  '' + (
+    if (cfg.paths == [ "-" ]) then
```

Comment:
You can avoid code duplication in the then.. and else.. branch by using something like
```
${if (cfg.Paths == ["-"]) then  (cfg.backupCommand + " | ") else "" }  borg create extraArgs \
```

Reply:
Should be solved in the update